src/zoning-district-class/zoning-district-class.service.spec.ts

```diff
@@ -2,10 +2,16 @@ import { Test } from "@nestjs/testing";
 import { ZoningDistrictClassRepositoryMock } from "test/zoning-district-class/zoning-district-class.repository.mock";
 import { ZoningDistrictClassService } from "./zoning-district-class.service";
 import { ZoningDistrictClassRepository } from "./zoning-district-class.repository";
-import { getAllZoningDistrictClassesQueryResponseSchema } from "src/gen";
+import {
+  getAllZoningDistrictClassesQueryResponseSchema,
+  getZoningDistrictClassesByIdQueryResponseSchema,
+} from "src/gen";
+import { ResourceNotFoundException } from "src/exception";
 
 describe("zoning district class service unit", () => {
   let zoningDistrictClassService: ZoningDistrictClassService;
+  const zoningDistrictClassRepositoryMock =
+    new ZoningDistrictClassRepositoryMock();
```

Comment:
Keep this here. It is the correct scope placement so that we can later access it in the tests. But..... (see later comment in test setup)